runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineConfiguration.java

```diff
@@ -359,7 +365,7 @@ private static int defaultStreamsBufferCapacity(
     private static int defaultEventsBufferCapacity(
         Configuration config)
     {
-        return ENGINE_BUFFER_SLOT_CAPACITY.get(config) * ENGINE_WORKER_CAPACITY.getAsInt(config);
+        return 4 * ENGINE_BUFFER_SLOT_CAPACITY.get(config);
```

Comment:
Let's default this to 4 * the default value of buffer slot capacity instead of changing it when buffer slot capacity changes.

I think making this change will fix the build breaking on the PR which is configuring the buffer slot small to test flow control and then running into issues with the events buffer being too small to log the event.